simulator/src/main/java/simblock/block/PoSpaceFoliage.java

```diff
@@ -0,0 +1,21 @@
+package simblock.block;
```

Comment:
This class and the PoSpace Trunk class are both unnecessary but I added them in to model the Chia approach. They may also be useful if we decide to make future changes